ihmc-high-level-behaviors/src/main/generated-java/us/ihmc/behaviors/activeMapping/ContinuousHikingParametersBasics.java

```diff
@@ -13,11 +13,6 @@ default void setEnableContinuousWalking(boolean enableContinuousWalking)
       set(ContinuousHikingParameters.enableContinuousWalking, enableContinuousWalking);
    }
 
-   default void setShortcutIsPressed(boolean shortcutIsPressed)
-   {
-      set(ContinuousHikingParameters.shortcutIsPressed, shortcutIsPressed);
```

Comment:
Old parameter time to remove

Reply:
This was originally made to show on the UI if the CTRL key was being held, however it never worked and the parameter files aren't meant to work like that so I removed it